### PR TITLE
navbar component

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**
Navbar:
-Simple navbar
-Navbar with searchbar

### CODE REVIEW

File: navigation/navigation.html [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+                        </ul>
+                    </div>
+
+                </div>
+
+            </div>
+
+            <div class="divider"></div>
+
+            <h3 class="intoduction--heading">Navbar with Searchbar</h3>
```
Comment (AshutoshM29) [inline navigation/navigation.html:167]: There is a typo in class name

**Review by AshutoshM29 — CHANGES_REQUESTED** [review]
Nice Work! you can merge it

File: navigation/navigation.html (hunk 1 above)
Comment (Adarsh-Patel-Dev) [reply navigation/navigation.html:167]: ok I'll change that

**Review by kotesh-arya — APPROVED** [review]
Looks good for a merge 👍🏻

File: navigation/navigation.html (hunk 1 above)
Comment (Adarsh-Patel-Dev) [reply navigation/navigation.html:167]: I'll change that typo after merging all branches into dev because there will be typo in every branch

**Pull request merged**